tests/vmi_configuration_test.go

```diff
@@ -1754,12 +1754,13 @@ var _ = Describe("Configurations", func() {
 
 			By("checking if default cache 'none' has been set to pvc disk")
 			Expect(disks[4].Alias.Name).To(Equal("hostpath-pvc"))
-			// PVC is mounted as tmpfs on kind, which does not support direct I/O.
-			// As such, it behaves as plugging in a hostDisk - check disks[6].
-			if tests.IsRunningOnKindInfra() {
-				Expect(disks[4].Driver.Cache).To(Equal(cacheWritethrough))
-			} else {
+                        Expect(disks[4].Source.File).ToNot(BeEmpty())
+			// Check if the underlying filesystem supports DIRECTIO.
+			// If it does, cache is None, else cache shoule be writethrough.
+			if tests.CheckDirectIOFlag(disks[4].Source.File) {
```

Comment:
I am not sure this check what you expect and if branching is as you intented.

Reply:
See my comment above. I will have to rethink the approach used.